packages/shared/src/components/image/Image.tsx

```diff
@@ -21,8 +23,13 @@ const ImageComponent = (
     }
   };
 
-  // eslint-disable-next-line jsx-a11y/alt-text
-  return <img {...props} ref={ref} onError={onError} />;
+  return (
+    <>
+      {isVideoType && <VideoPlayOverlay />}
```

Comment:
Now I understand why the other component has a wrapper, I think it should be done here rather than the `CardImage` component. Then we should utilize the `ConditionalWrapper` being enabled when it is a `videoType`.

Reply:
Second opinion:
To me the videoPlayOverlay has nothing to do with the generic image component.
Perhaps we should rather have a `VideoImage` component?

Else we have this logic spread everywhere as images are used widely and simply render a image with fallback basically.

Let me know what either of you think?

Reply:
In that case let's maybe go for the video image component?
It just moved the evaluation up and less prop drilling/polluting other components.

Reply:
Yep. Video component is the way to go. As long as we have everything in one place, I'm cool with it.